Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/HandJointServiceInspector.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System.Collections.Generic;
+using System;
+using UnityEditor;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [MixedRealityServiceInspector(typeof(HandJointService))]
+    public class HandJointServiceInspector : BaseMixedRealityServiceInspector
+    {
+        private const string ShowHandPreviewInSceneViewKey = "MRTK_HandJointServiceInspector_ShowHandPreviewInSceneViewKey";
+        private const string ShowHandJointFoldoutKey = "MRTK_HandJointServiceInspector_ShowHandJointFoldoutKey";
+        private const string ShowHandJointKeyPrefix = "MRTK_HandJointServiceInspector_ShowHandJointKeyPrefixKey_";
+        private static bool ShowHandPreviewInSceneView = false;
+        private static bool ShowHandJointFoldout = false;
+
+        private const float previewJointSize = 0.02f;
+        private static readonly Color previewJointColor = new Color(0.5f, 0.1f, 0.6f, 0.75f);
+        private static readonly Color enabledColor = GUI.backgroundColor;
+        private static readonly Color disabledColor = Color.Lerp(enabledColor, Color.clear, 0.5f);
+        private static Dictionary<TrackedHandJoint, bool> showHandJointSettings;
+        private static Dictionary<TrackedHandJoint, string> showHandJointSettingKeys;
+
+        //We want hand preview to always be visible
+        public override bool AlwaysDrawSceneGUI { get { return true; } }
+
+        public override void DrawInspectorGUI(object target)
+        {
+            IMixedRealityHandJointService handJointService = (IMixedRealityHandJointService)target;
+
+            EditorGUILayout.LabelField("Tracking State", EditorStyles.boldLabel);
+
+            if (!Application.isPlaying)
+            {
+                GUI.color = disabledColor;
+                EditorGUILayout.Toggle("Left Hand Tracked", false);
+                EditorGUILayout.Toggle("Right Hand Tracked", false);
+            }
+            else
+            {
+                GUI.color = enabledColor;
+                EditorGUILayout.Toggle("Left Hand Tracked", handJointService.IsHandTracked(Handedness.Left));
+                EditorGUILayout.Toggle("Right Hand Tracked", handJointService.IsHandTracked(Handedness.Right));
+            }
```

Comment:
I'm having a little trouble finding where to use this inspector - can you share some instructions on how to make it visible?

Reply:
If you're asking where can you see it in action, it exists on the hand joint service object created under the MRTK object. It's just like the pointer inspector.